include/dxc/DXIL/DxilConstants.h

```diff
@@ -1774,8 +1776,17 @@ enum class RayFlag : uint32_t {
   CullNonOpaque = 0x80,
   SkipTriangles = 0x100,
   SkipProceduralPrimitives = 0x200,
+  ForceOMM2State = 0x400, // Force 2-state in Opacity Micromaps
 };
 
+enum class RAYQUERY_FLAG : uint32_t {
+  RAYQUERY_FLAG_NONE = 0x00, // default
+  RAYQUERY_FLAG_ALLOW_OPACITY_MICROMAPS = 0x01,
+};
```

Comment:
This shouldn't be defined here.  These names are defined for HLSL and correspond to values defined by `RayQueryFlag` here.

Reply:
Removed entirely, I think it makes most sense to introduce this with the frontend change.

Reply:
On further investigation, it looks like this enum belongs in an external header, defined by the D3D team. D3D12_RAY_FLAGS is defined in `external\DirectX-Headers\include\directx\d3d12.h`, so I would think this enum would belong in that same file.